frontend/src/metabase/dashboard/components/DashCard.jsx

```diff
@@ -83,6 +83,14 @@ export default class DashCard extends Component {
         const isSlow = loading && _.some(series, (s) => s.duration) && (usuallyFast ? "usually-fast" : "usually-slow");
 
         const hasUnmappedParameters = _.any(series, (s) => s.json_query && _.any(s.json_query.parameters, (p) => p.target == null));
```

Comment:
Yeah I'm pretty sure they're supposed to do the same thing. Your implementation looks good, just remove `hasUnmappedParameters` and this should be fine.